src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/gui/CorrelationExportRuleGui.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jmeter.protocol.http.gui;
+
+import java.awt.BorderLayout;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.swing.JButton;
+import javax.swing.JFileChooser;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JTable;
+import javax.swing.table.DefaultTableCellRenderer;
+import javax.swing.table.TableModel;
+
+import org.apache.jmeter.gui.CorrelationTableModel;
+import org.apache.jmeter.gui.GuiPackage;
+import org.apache.jmeter.gui.util.FileDialoger;
+import org.apache.jmeter.protocol.http.correlation.CorrelationRule;
+import org.apache.jmeter.util.JMeterUtils;
+
+import net.minidev.json.JSONArray;
```

Comment:
Replace minidev by jackson. 
Minidev lib does not seem to be maintained anymore

Reply:
Replaced by jackson lib.

Reply:
Thank you we have replaced the Minidev library with Jackson library and checked horizontally and vertically for the
occurrences and updated them as well.